redash/query_runner/athena.py

```diff
@@ -78,6 +78,11 @@ def configuration_schema(cls):
                     'type': 'boolean',
                     'title': 'Use Glue Data Catalog',
                 },
+                'work_group': {
+                    'type': 'string',
+                    'title': 'Athena work group',
+                    'default': 'primary'
```

Comment:
Wouldn't it be safer/more backward compatible to just not pass anything if no value provided?

Reply:
According to the AWS [documentation](https://docs.aws.amazon.com/athena/latest/ug/user-created-workgroups.html):

> By default, each account has a primary workgroup and the default permissions allow all authenticated users access to this workgroup. The primary workgroup cannot be deleted.